packages/deployments/contracts/contracts/core/connext/helpers/RelayerProxyHub.sol

```diff
@@ -491,6 +470,42 @@ contract RelayerProxyHub is RelayerProxy {
     lastProposeAggregateRootAt = block.timestamp;
   }
 
+  /**
+   * @notice Wraps the `proposeAggregateRoot` function
+   * @dev This contract will validate the signer is a whitelisted proposer on the RootManager,
+   * and then call `propose` itself. This means this contract must *also* be whitelisted as a
+   * proposer on the RootManager.
+   * @param _snapshotId The snapshot id of the root to be proposed.
+   * @param _aggregateRoot The aggregate root to be proposed.
+   * @param _snapshotsRoots The roots of the connectors included in the aggregate.
+   * @param _domains The domains of the snapshots to be proposed.
+   * @param _signature Signature from the approved proposer.
+   */
+  function proposeAggregateRoot(
+    uint256 _snapshotId,
+    bytes32 _aggregateRoot,
+    bytes32[] calldata _snapshotsRoots,
+    uint32[] calldata _domains,
+    bytes memory _signature
+  ) external onlyRelayer nonReentrant {
+    if (!_proposeAggregateRootCooledDown()) {
+      revert RelayerProxyHub__proposeAggregateRootCooledDown_notCooledDown(
+        block.timestamp,
+        lastProposeAggregateRootAt + proposeAggregateRootCooldown
+      );
+    }
+
+    lastProposeAggregateRootAt = block.timestamp;
+
+    // Validate the signer
+    _validateProposeSignature(_snapshotId, _aggregateRoot, _signature);
+
+    // Propose the aggregate
+    rootManager.proposeAggregateRoot(_snapshotId, _aggregateRoot, _snapshotsRoots, _domains);
+
+    lastProposeAggregateRootAt = block.timestamp;
```

Comment:
duplicated assignment here and in line 498

Reply:
Done